### PR TITLE
update blockmarkup list from object to array

### PR DESCRIPTION
**Pull request by stevenwanuk**

### CODE REVIEW

File: src/main/java/com/bmj/content/model/overview/List.java [hunk 1]
```diff
@@ -39,8 +39,7 @@
 @XmlRootElement(name = "list")
 public class List {
 
-    @XmlElement(required = true)
-    protected Item item;
+    protected java.util.List<Item> item;
```
Comment (orenard) [inline src/main/java/com/bmj/content/model/overview/List.java]: Guys, did you not follow the instructions in the README to generate the Java code? Please do this ASAP and undo this change so that all code is generated in a consistent fashion!!!

Comment (stevenwanuk) [reply src/main/java/com/bmj/content/model/overview/List.java]: was generated from gradle:parse.xjc task, and talked to @orenard.
